crates/rooch-types/src/transaction/ethereum.rs

```diff
@@ -155,10 +154,7 @@ impl AbstractTransaction for EthereumTransactionData {
 
     fn authenticator_info(&self) -> Result<AuthenticatorInfo> {
         let chain_id = self.0.chain_id.ok_or(RoochError::InvalidChainID)?.as_u64();
-        let authenticator = Authenticator::new(
-            CoinID::Ether as u64,
-            self.into_signature()?.as_bytes().to_vec(),
-        );
+        let authenticator = Authenticator::new(1, self.into_signature()?.as_bytes().to_vec());
```

Comment:
We need to define an Enum for `1`. BuiltinAuthValidator::Ethereum?. We should keep the Ethereum signature in its original format.

Reply:
Defined and used Ethereum signature from `ethers` lib.